tests/Lewee.Application.Tests.Unit/TestWebApplicationFactory.cs

```diff
@@ -0,0 +1,96 @@
+using System.Reflection;
+using FreeMediator;
+using Lewee.Application;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+
+namespace Lewee.Application.Tests.Unit;
+
+/// <summary>
+/// Test web application factory for testing Lewee.Application behaviors
+/// </summary>
+public class TestWebApplicationFactory : WebApplicationFactory<Program>
```

Comment:
Use [TestServer](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.testhost.testserver?view=aspnetcore-9.0) instead of `WebApplicationFactory<T>`.

It allows you to setup you test server using specific services instead of setting up a test server based on a `Program` or `Startup` class.

Reply:
Therefore, this class is not needed.

A test of `TimedOperationBehavior` can add just that behaviour without adding other services and thus the test can isolate it's functionality in web server scenario.